README.md

```diff
@@ -1 +1 @@
-# NerdsOpposeSociety
\ No newline at end of file
+# NerdsOpposeSociety
```

Comment:
Again, should .md files be getting hit by prettier?

Reply:
I think it's good to have - Prettier handles a bunch of different file types & it's nice to keep them consistent

Reply:
Okay I guess I'm all good with it!